.github/workflows/validations.yaml

```diff
@@ -216,6 +216,7 @@ jobs:
 
       - name: Update PR benchmark results comment
         uses: marocchino/sticky-pull-request-comment@v2
+        continue-on-error: true
```

Comment:
if there is an error this should probably get advertised somehow

Reply:
the error output will be in the logs for troubleshooting if needed, however, we don't want to declare the benchmark validations as failed just because the posting of the comment on the PR fails.